server/static/staff/js/school_account/income_audit.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Created by liumengjun on 2016-09-27.
+ */
+$(function(){
+    paginationInit();
+    $('[data-action=switch_status]').click(function(e){
+        var $btn = $(this), $tr = $btn.closest('tr'),
+            status = $btn.attr('status'), recordId = $tr.attr('recordId');
+        var params = {'rid': recordId};
+        if (status != 'yes') {
```

Comment:
`params['action'] = 'mark_' + status;` and all following code can be simplified.